content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/roku.md

```diff
@@ -177,15 +198,15 @@ end sub
 
 ### Sample RUM sessions
 
-To control the data your application sends to Datadog RUM, you can specify a sampling rate for RUM sessions while [initializing the RUM Roku SDK][9] as a percentage between 0 and 100. You can specify the rate with the `sessionSampleRate` parameter.
+To control the data your application sends to Datadog RUM, you can specify a sampling rate for RUM sessions while [initializing the RUM Roku SDK][8] as a percentage between 0 and 100. You can specify the rate with the `sessionSampleRate` parameter.
```

Comment:
```suggestion
To control the data your application sends to Datadog RUM, you can specify a sampling rate for RUM sessions while [initializing the RUM Roku SDK][8]. You can specify the rate with the `sessionSampleRate` parameter as a percentage between 0 and 100.
```

Reply:
I don't know that all of my suggestions for fixing this type of language are consistent. Feel free to streamline them to the same format if you want. 😅